### ISSUE TITLE
REGRESSION: Purchase Email Subject ampersands getting improperly HTML encoded

### ISSUE THREAD

**Issue by markjaquith** [easydigitaldownloads/easy-digital-downloads#6828]
This was fixed in #4853, but regressed at some point. In EDD 2.9.6, ampersands are again being converted to `&amp;` in the sent emails. (To be clear, I don't know at what point in the last two years the issue regressed).

They look fine in the WordPress EDD purchase email settings page (i.e. they aren't TRIPLE encoded there).

**Comment by markjaquith** [comment]
Ah, it's just the test e-mail subject that is improperly encoded. Looks like that never got fixed.

**Comment by markjaquith** [comment]
The two pull requests above fix the issue for `master` and `release/3.0`.

**Comment by SDavisMedia** [comment]
Good catch. Definitely replicated on both `master` and `release/3.0`. Tested both PRs against both branches and confirmed fixed for both. Thanks!

All good for me, @cklosowski :+1:
